src/gps/GPS.h

```diff
@@ -76,13 +76,20 @@ class GPS : private concurrency::OSThread
     uint8_t fixType = 0;      // fix type from GPGSA
 #endif
   private:
-    const int serialSpeeds[6] = {9600, 115200, 38400, 4800, 57600, GPS_BAUDRATE};
+#if GPS_BAUDRATE_FIXED
+    // if GPS_BAUDRATE is specified in variant, only try that.
+    const int serialSpeeds[1] = {GPS_BAUDRATE};
+#else
+    const int serialSpeeds[3] = {9600, 115200, 38400};
+#endif
+    const int rareSerialSpeeds[3] = {4800, 57600, GPS_BAUDRATE};
```

Comment:
I think rare serial speeds should be inside of the else

Reply:
would fail to build due to the reference in gps.cpp

Reply:
I can modify the array though.